emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/write/AbstractDynamoDBRecordWriter.java

```diff
@@ -112,9 +112,11 @@ public void write(K key, V value) throws IOException {
     totolItemsWritten++;
 
     if (result != null) {
-      for (ConsumedCapacity consumedCapacity : result.getConsumedCapacity()) {
-        double consumedUnits = consumedCapacity.getCapacityUnits();
-        totalIOPSConsumed += consumedUnits;
+      if (result.getConsumedCapacity() != null) {
```

Comment:
Can you put this in the previous if statement?

Reply:
Is there not still a need to handle "unprocessedItems"?

Reply:
Oh whoops, I didn't look far enough down in the diff. disregard this